src/pages/account.tsx

```diff
@@ -0,0 +1,49 @@
+import React from "react"
+import { Router, RouteComponentProps } from "@reach/router"
+import { login, isAuthenticated, getProfile, logout } from "../utils/auth"
+import { Link } from "gatsby"
+import Layout from "../components/layout"
+
+type RouteComponent = React.FunctionComponent<RouteComponentProps>
+
+const Home: RouteComponent = () => <h3>Account Home</h3>
+const Settings: RouteComponent = () => <h3>Settings</h3>
+const Billing: RouteComponent = () => <h3>Billing</h3>
+
+const AccountPage = () => {
+  if (!isAuthenticated()) {
+    login()
+    return <p>Redirecting to login...</p>
+  }
+
+  const user = getProfile()
+
+  return (
+    <>
+      <Layout>
+        <nav>
+          <Link to="/">Home</Link> <Link to="/account">Account</Link>{" "}
+          <Link to="/account/billing">Billing</Link>{" "}
+          <Link to="/account/settings">Settings</Link>{" "}
+          <a
+            href="#logout"
+            onClick={e => {
+              logout()
+              e.preventDefault()
+            }}
+          >
+            Log Out
+          </a>
+        </nav>
+        <pre>Hello, {user.name}</pre>
```

Comment:
### 10. {Trigger the authentication flow}

> Lines {14} - {38}

This page should only render if the user is authenticated, so we redirect them to the login flow if they are not.

[Back →](https://github.com/erikdstock/gatsby-starter-ts-auth0/pull/2#pullrequestreview-239278342)